setup.py

```diff
@@ -92,12 +92,13 @@ def run(self):
 oss = ["oss2==2.6.1"]
 ssh = ["paramiko>=2.5.0"]
 hdfs = ["pyarrow>=0.17.0"]
+webdav = ["webdavclient3==3.14.5"]
```

Comment:
should we do >=?

Reply:
Yes, should be safe, the webdav3 API seems to be rather fixed. Just used what I got out of pip freeze...